RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -290,14 +293,20 @@
 initialStep = ClassifierMerger.clone()
 initialStep.inputClassifiers=['initialStepClassifier1','initialStepClassifier2','initialStepClassifier3']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(initialStep, initialStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorInitialStep_Phase1'),
+     src = 'initialStepTracks',
```

Comment:
this is redundant and was not present in the pre-DNN version

Reply:
Removed the redundant line.